packages/babel-plugin-transform-vite-meta-glob/src/__tests__/__snapshots__/index.ts.snap

```diff
@@ -64,6 +138,22 @@ const modules = {
 }
 
 
+`;
+
+exports[`vite-meta-glob glob some files normally, with { eager: false }: glob some files normally, with { eager: false } 1`] = `
+
+const modules = import.meta.glob("./fixtures/**/*{1,3}*", { eager: false })
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+import * as __glob__0_0 from './fixtures/file1.ts'
+import * as __glob__0_1 from './fixtures/file3.ts'
+const modules = {
+  './fixtures/file1.ts': __glob__0_0,
+  './fixtures/file3.ts': __glob__0_1
+}
```

Comment:
Oops, the `{ eager: false }` isn't quite working correctly. This should look like regular lazy-import. Taking a look

Reply:
Fixed